src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -2045,6 +2059,273 @@ def _pp_asym_gens_output_3ph(self):
+        #         pgm_load_profile["p_specified"] = np.append(np.nan)
+        #
+        #         pgm_load_profile["id"] = np.append(pgm_load_profile["id"], load_id_const_impedance)
+        #         pgm_load_profile["q_specified"] = np.append(
+        #         pgm_load_profile["q_specified"], q_mvar * const_z_multiplier)
+        #         pgm_load_profile["p_specified"] = np.append(np.nan)
+        #
+        #         pgm_load_profile["id"] = np.append(pgm_load_profile["id"], load_id_const_current)
+        #         pgm_load_profile["q_specified"] = np.append(
+        #         pgm_load_profile["q_specified"], q_mvar * const_i_multiplier)
+        #         pgm_load_profile["p_specified"] = np.append(np.nan)
+        #
+
+        # if "load.p_mw" not in pp_upd_data and "load.q_mvar" not in pp_upd_data:
+        #     return
+        #
+        # if "load.p_mw" in pp_upd_data and "load.q_mvar" in pp_upd_data:
+        #     p_mw_ids = np.array(pp_upd_data["load.p_mw"].columns)
+        #     q_mvar_ids = np.array(pp_upd_data["load.q_mvar"].columns)
+        #     # Should we sort the DF columns initially?
+        #     if p_mw_ids.sort() != q_mvar_ids.sort():
+        #         raise Exception("The IDs of load p_mw Datasource and load q_mvar Datasource are different!")
+        #
+        # if "load.p_mw" in pp_upd_data:
+        #     load_pmw_profile = pp_upd_data["load.p_mw"]
+        #     # Length of a DF represents time steps
+        #     time_steps = len(load_pmw_profile)
+        #     # Length of columns of a DF represents number of profiles
+        #     profiles = len(load_pmw_profile.columns)
+        #
+        #     pgm_load_profile = initialize_array("update", "sym_load", (time_steps, profiles))
+        #
+        #     pgm_load_profile["id"] = self._get_pgm_ids("load", np.array(load_pmw_profile.columns))
+        #
+        #     pgm_load_profile["p_specified"] = load_pmw_profile.to_numpy() * 1e6
+        #
+        #     if "load.q_mvar" in pp_upd_data:
+        #         pgm_load_profile["q_specified"] = pp_upd_data["load.q_mvar"].to_numpy() * 1e6
+        #
+        #     self.pgm_update_data["sym_load"] = pgm_load_profile
+        #
+        # if "load.q_mvar" in pp_upd_data and "load.p_mw" not in pp_upd_data:
+        #     load_qmvar_profile = pp_upd_data["load.q_mvar"]
+        #
+        #     time_steps = len(load_qmvar_profile)
+        #     profiles = len(load_qmvar_profile.columns)
+        #
+        #     pgm_load_profile = initialize_array("update", "sym_load", (time_steps, profiles))
+        #
+        #     pgm_load_profile["id"] = self._get_pgm_ids("load", np.array(load_qmvar_profile.columns))
+        #
+        #     pgm_load_profile["q_specified"] = load_qmvar_profile.to_numpy() * 1e6
+        #
+        #     self.pgm_update_data["sym_load"] = pgm_load_profile
+
+    # pylint: disable-msg=too-many-locals
+    def _pp_update_sgens(self):  # pragma: no cover
```

Comment:
todo: probably a good idea to split in separate functions if it complains about this

(note: i understand why it's still in there, so no hurry. i just add this comment for reference)